api/src/main/java/org/openmrs/api/impl/ProgramServiceValidatorImpl.java

```diff
@@ -0,0 +1,25 @@
+/**
+ * This Source Code Form is subject to the terms of the Mozilla Public License,
+ * v. 2.0. If a copy of the MPL was not distributed with this file, You can
+ * obtain one at http://mozilla.org/MPL/2.0/. OpenMRS is also distributed under
+ * the terms of the Healthcare Disclaimer located at http://openmrs.org/license.
+ *
+ * Copyright (C) OpenMRS Inc. OpenMRS is a registered trademark and the OpenMRS
+ * graphic logo is a trademark of OpenMRS Inc.
+ */
+
+package org.openmrs.api.impl;
+
+import org.openmrs.validator.ProgramServiceValidator;
+import org.openmrs.PatientProgram;
+import org.openmrs.api.APIException;
+import org.springframework.stereotype.Component;
+
+@Component
+public class ProgramServiceValidatorImpl implements ProgramServiceValidator {
+    
+    @Override
+    public void validate(PatientProgram patientProgram) throws APIException {
+        // default constructor
```

Comment:
What does the above comment mean?

Reply:
I was trying out to see if it will solve the codacy error for method missing document, I'm changing it right away.